config.json

```diff
@@ -180,6 +180,12 @@
       "values": [],
       "defaultValue": ""
     },
+    "StrictPrerequisiteVersionMatchAdditionalArg": {
```

Comment:
Yes, prerequisite is hard to type.  Missing one of the "i" is easy.  Will rename as tool or tools as appropriate.